pkg/parse/parse.go

```diff
@@ -535,6 +535,8 @@ func defaultIsolation() buildah.Isolation {
 	isolation := os.Getenv("BUILDAH_ISOLATION")
 	if strings.HasPrefix(strings.ToLower(isolation), "oci") {
 		return buildah.IsolationOCI
+	} else if strings.HasPrefix(strings.ToLower(isolation), "chroot") {
+		return buildah.IsolationChroot
```

Comment:
Should we throw an error if environment is set but not set to chroot or oci>

Reply:
Yes, we should.  Fixing.